virtio-devices/src/iommu.rs

```diff
@@ -680,8 +683,8 @@ impl IommuEpollHandler {
         paused_sync: Arc<Barrier>,
     ) -> result::Result<(), EpollHelperError> {
         let mut helper = EpollHelper::new(&self.kill_evt, &self.pause_evt)?;
-        helper.add_event(self.queue_evts[0].as_raw_fd(), REQUEST_Q_EVENT)?;
-        helper.add_event(self.queue_evts[1].as_raw_fd(), EVENT_Q_EVENT)?;
+        helper.add_event(self.request_queue_evt.as_raw_fd(), REQUEST_Q_EVENT)?;
+        helper.add_event(self.event_queue_evt.as_raw_fd(), EVENT_Q_EVENT)?;
```

Comment:
Given we're not implementing the event queue at all, there's no need for registering the eventfd either. In a separate patch, could you please remove this line, and therefore change `event_queue_evt` into `_event_queue_evt` in the `IommuEpollHandler` struct?

Reply:
Sounds good.